config.json

```diff
@@ -85,7 +85,8 @@
     "list-ops",
     "diamond",
     "all-your-base",
-    "isogram"
+    "isogram",
+    "tournament"
```

Comment:
The 'problems' key has been removed from the config.json file, so this now causes a conflict.